Editor/MenuActions/Editors/NewPolyShape.cs

```diff
@@ -34,6 +34,23 @@ public override bool enabled
 
         public override ActionResult DoAction()
         {
+            var inspWindow = EditorWindow.GetWindow<InspectorWindow>();
```

Comment:
This will instantiate a new Inspector window if one is not already open. This should just look for an existing window.

Reply:
Indeed good catch.